instrumentation/otel-webserver-module/src/apache/ApacheHooks.cpp

```diff
@@ -516,6 +520,52 @@ void fillRequestPayload(request_rec* request, appd::core::RequestPayload* payloa
     val = request->method ? request->method: " ";
     payload->set_http_request_method(val);
 
+    // websrv-698 Setting server span attributes
+    payload->set_status_code(request->status);
+
+    if (request->server)
+    {
+        payload->set_server_name(request->server->server_hostname);
+    }
+    payload->set_scheme(ap_run_http_scheme(request));
```

Comment:
ap_ functions are like hooks many a times. Is there any documentation for the function ap_run_http_scheme?

Reply:
/**
 * Get the scheme for a request.
 * @param r The request
 */
#define ap_http_scheme(r)       ap_run_http_scheme(r)